test/recipes/15-test_genrsa.t

```diff
@@ -105,18 +102,14 @@ ok(!run(app([ 'openssl', 'genpkey', '-propquery', 'unknown',
    "genpkey requesting unknown=yes property should fail");
 
 
- SKIP: {
-    skip "Skipping rsa command line test", 4 if disabled("deprecated-3.0");
-
-    ok(run(app([ 'openssl', 'genrsa', '-3', '-out', 'genrsatest.pem', $good ])),
```

Comment:
Err.. the old tests were testing the -3 option.. Now nothing probably does this.

Reply:
Scroll up :)  This is testing a minimal sized key with `-3`.

Reply:
Put this back for deprecated builds only.